src/sdk/model/company/support-ticket/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './__json__/index';
```

Comment:
remove this file. (since that last refactoring PR was merged, theres no need to add index.ts files anymore)